pkg/webhook/util/convertor/util.go

```diff
@@ -0,0 +1,50 @@
+package convertor
+
+import (
+	"strconv"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/validation"
+	"k8s.io/kubernetes/pkg/apis/core"
+	corev1 "k8s.io/kubernetes/pkg/apis/core/v1"
+)
+
+func ConvertPodTemplateSpec(template *v1.PodTemplateSpec) (*core.PodTemplateSpec, error) {
+	coreTemplate := &core.PodTemplateSpec{}
+	if err := corev1.Convert_v1_PodTemplateSpec_To_core_PodTemplateSpec(template.DeepCopy(), coreTemplate, nil); err != nil {
+		return nil, err
+	}
+	return coreTemplate, nil
+}
+
+func ConvertCoreVolumes(volumes []v1.Volume) ([]core.Volume, error) {
+	coreVolumes := []core.Volume{}
+	for _, volume := range volumes {
+		coreVolume := core.Volume{}
+		if err := corev1.Convert_v1_Volume_To_core_Volume(&volume, &coreVolume, nil); err != nil {
+			return nil, err
+		}
+		coreVolumes = append(coreVolumes, coreVolume)
+	}
+	return coreVolumes, nil
+}
+
+func GetPercentValue(intOrStringValue intstr.IntOrString) (int, bool) {
```

Comment:
Do you plan to expose this method? If not, renaming it to getPercentValue would be better.

Reply:
This method may be used outside this method, although not yet